apps/blockchain/test/support/byzantium_failing_tests.txt

```diff
@@ -0,0 +1,4936 @@
+GeneralStateTests/stArgsZeroOneBalance/addNonConst_d0g0v0.json
```

Comment:
Yeah, we thought about this question, and this is where we landed: we have 25 tests passing, and it is nice to keep track of the progress as we improve byzantium (just like we've been doing with other hardforks). Since they are a lot of tests, we didn't want to just copy/paste them from the `generate_blockchain_tests` script, so we modified the script to automatically populate this file and the `blokchain_tests` just read this file. 

I also think that not running these tests opens us up to the possibility of introducing regression bugs while we work on Byzantium. So it gives us the added confidence of seeing when we pass tests and when we may be causing some of the ones that were passing to fail. 

What do you think?